src/function-configuration.ts

```diff
@@ -64,6 +73,6 @@ export interface RuntimeOptions {
 }
 
 export interface DeploymentOptions extends RuntimeOptions {
-  regions?: string[];
+  regions?: Array<typeof SUPPORTED_REGIONS[number]>;
```

Comment:
This change is from another PR we merged right?

Reply:
Right, it's no longer in the diff after merging master.